packages/slice-machine/src/features/navigation/RepositoryInfo.tsx

```diff
@@ -0,0 +1,43 @@
+import {
+  Box,
+  IconButton,
+  Text,
+  Tooltip,
+  useMediaQuery,
+} from "@prismicio/editor-ui";
+
+import { useRepositoryInformation } from "@/hooks/useRepositoryInformation";
+
+export function RepositoryInfo() {
+  const { repositoryName, repositoryDomain, repositoryUrl } =
+    useRepositoryInformation();
+  const isCollapsed = useMediaQuery({ max: "medium" });
+
+  return (
+    <Box
+      justifyContent={isCollapsed ? "center" : "space-between"}
+      padding={{ bottom: isCollapsed ? 16 : 0 }}
```

Comment:
asymmetrical component

Reply:
yes, it became asymmetrical when collapsed, but thanks to it the height of that element is maintained, so the rest of the sidebar below that doesn't jump up and down (in the collapsed view, there is only icon visible)

Reply:
maybe we should use height instead then? or rendering an icon that takes the same amount of space so when the text is not there there's no layout shift

Reply:
Changed to fixed height.
Also thanks to it spotted the difference that I overlooked: in old version height was fixed, in my (before this fix) if repo name would be long it would wrap and cause to increase height of that component.
Should be good now.